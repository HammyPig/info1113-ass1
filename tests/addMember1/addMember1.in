ADD MEMBER poopoo
add member poo poo pee pee
aDd mEmber pee pee poopoo