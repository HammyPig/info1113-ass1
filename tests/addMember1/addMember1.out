user: Success.

user: Success.

user: Success.

user: Ending Library process.
